components/embl-logo/embl-logo.scss

```diff
@@ -1,9 +1,13 @@
 // embl-logo
 
 .embl-logo {
+  background-image: url('../assets/embl-logo/assets/logo.png');
```

Comment:
`.png` here?

Reply:
Forgot to remove some of the PNG stuff, should be sorted now.